source/mainnet/net/voting/guardians.rst

```diff
@@ -0,0 +1,193 @@
+.. _guardians:
+.. include:: ../../variables.rst
+
+=========
+Guardians
+=========
+
+The :term:`guardian` (via the guardian application) participates in all three phases of the election: the :term:`setup phase` before the election, they may cast votes during the :term:`election phase`, and the :term:`tally phase` at the end of the election.
+
+The setup phase ensures that the election tallying is decentralized. A number of guardians as defined by the election parameters in the smart contract register their public keys in the smart contract to say that they will :term:`tally` election votes. After the election closes, the guardians :term:`decrypt their share of the votes<decryption share>` and send the decrypted tally back to the smart contract. In this way, no one party tallies the votes for the election, ensuring privacy of the votes. Correctness of the result follows from the zero-knowledge proofs generated by the guardians to prove that they performed all steps correctly.
+
+.. note::
+
+    Guardians cannot use the Desktop wallet or |mw-gen1| because these wallets cannot export account keys.
```

Comment:
CryptoX cannot export account keys either, can it? Or am I just blind when looking through the app? 😄